pylab/accounts/models.py

```diff
@@ -0,0 +1,16 @@
+import django.contrib.auth.models as auth_models
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(auth_models.User)
```

Comment:
[related_name='profile'](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ForeignKey.related_name) would be more convenient, since you will be able to call `user.profile` instead of `user.userprofile`.